firehose.c

```diff
@@ -130,13 +130,13 @@ static int firehose_read(struct qdl_device *qdl, int timeout_ms,
 	timeradd(&now, &delta, &timeout);
 
 	for (;;) {
-		n = qdl_read(qdl, buf, sizeof(buf), 100);
+		n = qdl_read(qdl, buf, sizeof(buf), 700);
```

Comment:
But when we hit this timeout, won't timercmp() below result in the loop to be continued?

Reply:
Please see pull request #67.